app/views/data_tables/index.html.haml

```diff
@@ -11,5 +11,10 @@
       - table_column = model_param + '.' + table_column unless table_column.include? '.'
       %th= t("tables.#{table_column}")
 
+  - if model_class.respond_to? :count_columns
+    - model_class.count_columns.each do |count_column|
+      - count_column = count_column.scan(/\(([^\)]+)\)/).last.first
```

Comment:
To get only the content inside parentheses (). So from count(a.b) only a.b is taken, in order to deliver a proper join.